src/tools/idlpp/src/mcpp_out.h

```diff
@@ -13,10 +13,10 @@
 #define MCPP_OUT_H
 
 typedef enum {
-  OUT,
-  ERR,
-  DBG,
+  MCPP_OUT,
+  MCPP_ERR,
+  MCPP_DBG,
   NUM_OUTDEST
```

Comment:
Not `MCPP_NUM_OUTDEST` for consistency?

Reply:
Bummer, that's what you get for automating source-to-source transformations on a C code base ... This is too terrible, I'll fix this before merging it.